tests/rptest/scale_tests/large_messages_test.py

```diff
@@ -266,13 +281,17 @@ def _get_samples(name, label="kafka"):
 
     @cluster(num_nodes=7, log_allow_list=RESTART_LOG_ALLOW_LIST)
     @matrix(
-        message_size_mib=[8, 16],
+        message_size_mib=[8, 16, 32],
         apply_throughput_limits=[False, True],
         mode=[Mode.MANY_PARTS, Mode.TEN_TOPICS],
+        default_consumer_config=[False, True],
```

Comment:
Note that the additional parameter here increases the test runtime from ~1hr to ~2hr.